src/main/java/com/busify/busifyappproject/model/Passenger.java

```diff
@@ -0,0 +1,42 @@
+package com.busify.busifyappproject.model;
+
+import javax.persistence.*;
```

Comment:
Avoid using star imports since we are not aware of what we are using in the code. Set up your IntelliJ environment not to replace imports with star imports: https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports